huggingface/pytorch/tgi/docker/2.2.0/Dockerfile

```diff
@@ -211,6 +211,7 @@ RUN apt-get update && apt-get upgrade -y && DEBIAN_FRONTEND=noninteractive apt-g
     git \
     libexpat1 \
     libgssapi-krb5-2 \
+    linux-libc-dev \
     && rm -rf /var/lib/apt/lists/*
 
 # Copy conda with PyTorch installed
```

Comment:
Do we need to also update `releases.json`?

Reply:
thanks for catching that, my branch was still on a version that had the correct releases.json, fixed now